.ameba.yml

```diff
@@ -1,13 +0,0 @@
-# This configuration file was generated by `ameba --gen-config`
-# on 2018-06-19 12:29:03 -05:00 using Ameba version 0.7.0.
-# The point is for the user to remove these configuration records
-# one by one as the reported problems are removed from the code base.
-
-# Problems found: 2
-# Run `ameba --only PredicateName` for details
-PredicateName:
-  Description: Disallows tautological predicate names
```

Comment:
I'm assuming this config is now disallowed by default?

Reply:
https://github.com/veelenga/ameba/commit/970ca4be1b93068694bd0cc99ce2ef543e70bfae